packages/Takeover/src/components/Header/Header.styles.js

```diff
@@ -28,10 +28,32 @@ export const Header = styled.div(
 );
 
 export const Heading = styled(OriginalHeading)`
-  margin: 0 0 0 ${spacer(2)};
+  margin: 0;
+  padding: 18px 0 17px ${spacer(3)};
 `;
 
 export const CloseButton = styled.div`
   border-left: 1px solid ${tokens.border.color};
-  padding: ${spacer(1)};
+  padding: 16.5px ${spacer(2)} ${spacer(2)} 15.5px;
```

Comment:
Is there any difference if we just say `padding: ${spacer(2)};`

Reply:
the total height won't change but yes it's 0.5 px different in terms of spacing. Do you want me just set it as 16px instead?

Reply:
yes please 😄